change/@fluentui-react-charting-2020-10-26-12-57-10-lineChartTests.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```

Since this is just tests
```suggestion
  "type": "none",
```